frontend/src/components/landing-page.tsx

```
@@ -1,412 +1,378 @@
+
+const HackathonCard: React.FC<{ hackathon: Hackathon, delay: number }> = ({ hackathon, delay }) => {
+  const [isVisible, setIsVisible] = useState(false)
+
+  useEffect(() => {
+    const timer = setTimeout(() => setIsVisible(true), delay)
+    return () => clearTimeout(timer)
+  }, [delay])
+
+  if (!hackathon || !hackathon.attributes) return null;
+
+  const { Title, Theme, Date, Location, EventStatus } = hackathon.attributes;
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)